msnoise/default.py

```diff
@@ -47,7 +50,7 @@
 default['pws_power'] = ["If stack_method='pws', Power of the Weighting: 2.0 "
                         "",'2.0', float]
 
-default['crondays'] = ["Number of days to monitor with cron [-1], must be a negative number", '-1']
+default['crondays'] = ["Number of days to monitor with cron [1]", '1']
```

Comment:
you can make it 
``["Number of days to monitor with cron [1]", '1', int]``

so this value gets automatically converted to int when using ```get_config``

Reply:
Oh, you're right! In fact I realised this possibility only a few days ago,, after I pushed these changes. That would have been useful indeed, but I am finally keeping it as a string to use the format I propose below.